### ISSUE TITLE
Does electrum really need home access?

### ISSUE THREAD

**Issue by black-puppydog** [TPaw/TPawSnaps#2]
given that the user can (and should!) remember their seed, is that permission really necessary?

**Comment by tomascw** [comment]
The software needs to store the user's information somewhere.

**Comment by black-puppydog** [comment]
Yes, but that doesn't necessarily require access to the entire home. And also, that permission does *not* include hidden (dotted) files.
Without the home interface, electrum should still have access to `/home/$USER/snap/<snap>/common` for storing information *across* snap versions, as well as `/home/$USER/snap/<snap>/<revision>` for per-version data, see [the docs](https://snapcraft.io/docs/reference/confinement)

It would be cool to drop the home interface, but it's definitely a bonus :P

**Comment by tomascw** [comment]
That I'm aware of but I'm not sure if it's possible to change the data directory with Electrum.
